src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/UpgradeAssistantCommand.cs

```diff
@@ -23,6 +23,7 @@ public UpgradeAssistantCommand(string name)
             AddOption(new Option<bool>(new[] { "--ignore-unsupported-features" }, LocalizedStrings.UpgradeAssistantCommandIgnoreUnsupported));
             AddOption(new Option<DirectoryInfo>(new[] { "--vs-path" }, LocalizedStrings.UpgradeAssistantCommandVS));
             AddOption(new Option<DirectoryInfo>(new[] { "--msbuild-path" }, LocalizedStrings.UpgradeAssistantCommandMsbuild));
+           /* AddOption(new Option<string>(new[] { "--format", "--f" }, LocalizedStrings.UpgradeAssistantCommandFormat));*/
```

Comment:
why is this commented? Don't you want to expose that?

Reply:
Removed the commented-out line and moved this function call to ConsoleAnalyzeCommand.cs as formatting only falls under analysis for now.